BaseTools/Scripts/PatchCheck.py

```diff
@@ -304,12 +304,49 @@ def __init__(self, diff):
         self.line_num = 0
         self.state = START
         self.new_bin = []
+        self.LicenseCheck(self.lines, self.count)

```

Comment:
Python code style uses snake case for functions and camel case for classes. The rest of the code seems to be adhering to it. This looks like a class construction.